hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -202,6 +202,10 @@ public Operation convert(String value) throws ParameterException {
         + " to break ties between records with same key in input data. Default: 'ts' holding unix timestamp of record")
     public String sourceOrderingField = "ts";
 
+    @Parameter(names = {"--source-delete-field"}, description = "Field within source record to decide"
+            + " is this record is deleted. Default: " + OverwriteWithLatestAvroPayload.DEFAULT_DELETE_FIELD)
```

Comment:
this leakage is what I wish to avoid. Why would these classes access OverwriteWithLatestAvroPayload. moving it to DatasourceUtils or some config classes makes sense.